timesketch/api/v1/utils.py

```diff
@@ -41,6 +43,32 @@ def bad_request(message):
     return response
 
 
+def get_sketch_attributes(sketch):
+    """Returns a list of attributes of a sketch."""
+    attributes = []
+    ontology_def = ontology.ONTOLOGY
+    for container in sketch.attributes:
+        if container.sketch_id != sketch.id:
```

Comment:
When would this happen? Or is this just a safety measure?

Reply:
safety measure... shouldn't happen